cfme/tests/storage/test_volume.py

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8 -*-
+import pytest
+from widgetastic.exceptions import NoSuchElementException
+
+from cfme import test_requirements
+from cfme.cloud.provider import CloudProvider
+from cfme.cloud.provider.openstack import OpenStackProvider
+from cfme.storage.volume import Volume
+
+from utils import testgen
+from utils.appliance.implementations.ui import navigate_to
+
+
+pytest_generate_tests = testgen.generate([CloudProvider])
```

Comment:
I like that construction @izapolsk - actually found it recently and recommended to Jeff for proxy tests generation.

At this time I won't be expanding the storage tests, though I have a feeling it may end up falling back to me eventually!